Home.md

```diff
@@ -17,3 +17,73 @@ Use the [`#WindowsCommunityToolkit`](https://twitter.com/search?q=%23WindowsComm
+
+Shrestha Files is a modern and minimalist dual pane file manager with tabs and color themes support.
+
+- It uses GridSplitter in dual pane mode to resize two panes.
+- It uses Guard to validate arguments.
+- It uses MVVM toolkit and is gradually moving to MVVM-ify more! All settings pages and navigation bar edit page mostly follow MVVM pattern.    
+- It uses EnumValuesExtensions since the last version to load enum values in SplitButton options for view type and sort type.
+
+> Here's a quick overview of: [Shretha Files](https://youtu.be/-LBXu0lzh_Y "https://youtu.be/-LBXu0lzh_Y"). Find the App [here](https://www.microsoft.com/p/shrestha-files-pro-a-modern-dual-panel-file-manager-with-tabs-and-colors/9npnffsv2hqm)
+
+<h1 align="center">Yugen Mosaic</h1>
+
+![YugenMosaic](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic.png) 
+![YugenMosaic2](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/YugenMosaic2.png)
+
+>Creator: [PandaSharp](https://github.com/Panda-Sharp)
+
+Yugen Mosaic is a free app that creates digital art pictures based on your own images. With it, you can use an unlimited amount of images, to create a photo mosaic which is an image composed of many tiled photos. Yugen is an awareness of the universe that triggers emotional responses too deep and powerful for words, and that is just what your art will do with a little help from Yugen Mosaic!
+
+Find the App in [Microsoft Store](https://www.microsoft.com/en-us/p/yugen-mosaic/9pf0s24cx0d4). More information regarding this [project](https://github.com/Panda-Sharp/Yugen.Mosaic)
+
+<h1 align="center">OneLocker and Legere</h1>
+
+![Onelocker-legere](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Onelocker-Legere.png)
+
+>Creator: [Sergio Pedri](https://github.com/Sergio0694)
+
+**OneLocker:** is a UWP password manager with the goal of being as feature complete as possible. It supports tons of UWP-specific features (eg. Windows Hello, Timeline, live tiles, notifications, background sync, Windows Ink, Surface Dial, etc.) and the database is heavily customizable. The creator have used the Toolkit here for several UWP helpers (eg. SystemInfo) and extensions.
+
+**Legere:** is a fully customizable UWP client for Reddit supporting multiple users, live tiles, myTube! integration, and more. 
+
+**According to the Creator:**
+
+> ***I'm using several Toolkit controls like the WrapPanel to display post tags, as well as lots of helpers. I'm currently in the process of migrating the codebase to the Toolkit 7.0***
+
+<h1 align="center">Console/IDE</h1>
+
+![Console/IDE](https://github.com/Kyaa-dost/WindowsCommunityToolkit-wiki/blob/App-Gallery/images/Console:IDE.png)
+
+>Creator: [Sergio Pedri](https://github.com/Sergio0694)
+
+Console/IDE: is a fully featured console and IDE/debugger for an obscure programming language.
+
+**In the words of Sergio**: 
```

Comment:
We should be consistent about how we quote the creators for each app. So we should be like:

```
### [App Name](Link to store) by [Creator](Link to GitHub Profile)
(Image) + link to Store/GitHub
App Description
> "Quote from Developer" - _Developer Name_
Toolkit Components Used: MVVM Toolkit, WrapPanel, etc...
[GitHub](link) - [Store](link)
```